frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -19,6 +21,8 @@
 public class WorkerLifeCycle {
 
     private static final Logger logger = LoggerFactory.getLogger(WorkerLifeCycle.class);
+    private static final Pattern PATTERN =
```

Comment:
Can we give more descriptive name to `PATTERN`?

Reply:
Done